ImageInLib/ImageInLib/src/segmentation3D_subsurf.c

```diff
@@ -315,9 +316,9 @@ bool subsurfSegmentationTimeStep(dataType **prevSol_extPtr, dataType **gauss_sei
 			}
 		}
 	} while (mean_square_residue > segParameters.gauss_seidelTolerance && z < segParameters.maxNoGSIteration);
-	printf("The number of iterations is %zd\n", z);
+	//printf("The number of iterations is %zd\n", z);
 	//printf("Residuum is %e\n", mean_square_residue);
-	printf("Step is %zd\n", segParameters.numberOfTimeStep);
+	//printf("Step is %zd\n", segParameters.numberOfTimeStep);
```

Comment:
if you need this output for debugging purposes, please "control it" by defined constants